SleepEngine/Engine/Utils/unit_utils.h

```diff
@@ -0,0 +1,27 @@
+// Copyright 2019, Tihran Katolikian
+#include <Engine/Config/EngineConfig.h>
+#pragma once
+
+BEGIN_NAMESPACE_SLEEP
+
+inline float metersToPixels(float meters)
+{
+    return meters * globalEngineConfig()->getData().MeterLengthInPixels / 2.f;
```

Comment:
this is so sad that this values are runtime dependent now. It cannot be optimized away :(

Reply:
ah, performance'll manage

Reply:
now it will read from static memory now)
hm maybe things like meter length to pixels should be hardcoded?

Reply:
Not hardcoded by I guess it should be resolved on compile time.
Lets keep it like this, performance is not out main problem now